website/blog/2025-01-16-3.5.0.md

```diff
@@ -0,0 +1,330 @@
+
+- Support for the new `objectWrap` option
+- Support for the new experimental `experimentalOperatorPosition` option
+- Support for TypeScript configuration file
+
+See each section for details.
+
+<!-- truncate -->
+
+If you appreciate Prettier and would like to support our work, please consider sponsoring us directly via [our OpenCollective](https://opencollective.com/prettier) or by sponsoring the projects we depend on, such as [typescript-eslint](https://opencollective.com/typescript-eslint), [remark](https://opencollective.com/unified), and [Babel](https://opencollective.com/babel). Thank you for your continued support!
+
+## Highlights
+
+### JavaScript
+
+#### Add experimental option for breaking lines before binary operators ([#7111](https://github.com/prettier/prettier/pull/7111) by [@btmills](https://github.com/btmills))
+
+This is implemented behind the `--experimental-operator-position <start|end>` flag.
+
+When binary expressions wrap lines, `start` prints the operators at the start of new lines. Placing binary operators at the beginning of wrapped lines can make the operators more prominent and easier to scan.
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+var a = Math.random() * (yRange * (1 - minVerticalFraction)) + minVerticalFraction * yRange - offset;
+
+// `experimentalOperatorPosition: end` (default behavior)
+var a =
+  Math.random() * (yRange * (1 - minVerticalFraction)) +
+  minVerticalFraction * yRange -
+  offset;
+
+// `experimentalOperatorPosition: start`
+var a =
+  Math.random() * (yRange * (1 - minVerticalFraction))
+  + minVerticalFraction * yRange
+  - offset;
+```
+
+#### Implement `objectWrap` config option ([#16163](https://github.com/prettier/prettier/pull/16163) by [@pauldraper](https://github.com/pauldraper), [@sosukesuzuki](https://github.com/sosukesuzuki))
+
+Prettier has historically done semi-manual formatting of multi-line JavaScript object literals.
+
+Namely, an object is kept on multiple lines if there is a newline prior to the first property, even if it could fit on a single line. See [Multi-line objects](rationale.md#multi-line-objects) for more details.
+
+While this behavior continues to be the default, `--object-wrap=collapse` instead ignores whitespace when formatting object literals.
+
+<!-- prettier-ignore -->
+```js
+// Input
+const obj1 = {
+  name1: "value1", name2: "value2",
+};
+
+const obj2 = { name1: "value1",
+  name2: "value2",
+};
+
+// Prettier 3.4
+const obj1 = {
+  name1: "value1",
+  name2: "value2",
+};
+
+const obj2 = { name1: "value1", name2: "value2" };
+
+// Prettier 3.5 (with `--object-wrapping=collapse`)
+const obj1 = { name1: "value1", name2: "value2" };
+
+const obj2 = { name1: "value1", name2: "value2" };
+```
+
+### API
+
+#### Add support for TypeScript config files ([#16828](https://github.com/prettier/prettier/pull/16828) by [@itsyoboieltr](https://github.com/itsyoboieltr) & [@fisker](https://github.com/fisker))
```

Comment:
Maybe this worth highlight?

BTW, you don't need use separate PR to highlight entries, it can be done when preparing blog post.

Reply:
> BTW, you don't need use separate PR to highlight entries, it can be done when preparing blog post.

I was thinking the same thing, but in our release flow, after clearing the `changelog_unureleased`, we would have to merge this PR, which would cause conflicts. To avoid this, I separate PRs